### PR TITLE
Friesian: support backend choice in workflow examples

### PR DESCRIPTION
**Pull request by hamham223**
### 1. Why the change?

As title, user can choose backend via program args `--backend` when running the example

### 2. How to test?

- [ ] Fresian-Example-Test

### CODE REVIEW

File: python/friesian/example/wnd/recsys2021/wnd_train_recsys.py [hunk 1]
```diff
@@ -262,6 +262,8 @@ def label_cols(self):
     parser = OptionParser()
     parser.add_option('--cluster_mode', type=str, default="local",
                       help='The cluster mode, such as local, yarn or standalone.')
+    parser.add_option('--backend', type=str, default="ray",
+                      help='The backend of Orca Estimator, either ray or spark.')
```
Comment (hamham223) [inline python/friesian/example/wnd/recsys2021/wnd_train_recsys.py:266]: `add_option` does not support `choices`

File: python/friesian/example/two_tower/train_2tower.py [hunk 2]
```diff
@@ -184,22 +188,23 @@ def organize_cols(tbl):
...
-                               init_ray_on_spark=True)
+                               init_ray_on_spark=ray_on_spark)
     elif args.cluster_mode == "yarn":
         sc = init_orca_context("yarn-client", cores=args.executor_cores,
                                num_nodes=args.num_executors, memory=args.executor_memory,
                                driver_cores=args.driver_cores, driver_memory=args.driver_memory,
                                conf=spark_conf, extra_python_lib="model.py",
                                object_store_memory="80g",
-                               init_ray_on_spark=True)
+                               init_ray_on_spark=ray_on_spark)
```
Comment (hkvision) [inline python/friesian/example/two_tower/train_2tower.py:207]: Can directly remove this argument

**Comment by hamham223** [comment]
Do we have other examples that need to add the `backend` option?

**Review by hkvision — APPROVED** [review]

**Comment by hkvision** [comment]
> Do we have other examples that need to add the `backend` option?

Maybe not for now.

**Pull request merged**